docs/user-guide/release-notes/2.9.x.rst

```diff
@@ -14,4 +14,7 @@ New Features
   parsed on upload. ``pulp-admin`` also has upload, remove, and search support for 
   package_langpacks.
 * The yum distributor now uses the configured checksum type for all repo metadata.
-
+* The export distributor now uses the configured checksum type for all repo metadata. The pulp-admin
```

Comment:
I would put the note about yum and export distributor under same bullet because later you mention under export distributor 'both distributors' and it is not very clear what are the both distributors you are talking about:
* The yum and export distributors now use the configured checksum type for all repo metadata. The pulp-admin
   repo "create" and "update" commands now save the specified checksum type on both distributors.